### ISSUE TITLE
Arrow Style for Directed Graphs

### ISSUE THREAD

**Issue by Livnetata** [robert-haas/gravis#7]
Is it possible to change the arrow style in directed graphs (something like adding an attribute for "arrowstyle", as is implemented in matplotlib https://matplotlib.org/stable/api/_as_gen/matplotlib.patches.ArrowStyle.html)?

Thanks,
Neta

**Comment by ThomasKimber** [comment]
Yes please, I came here to request the same thing. I'm interested in modelling entity relationship diagrams, where such endings might represent one/many/optional relationship cardinalities, and where I previously explored this in SVG, you can attach marker-end and marker-start decorations which opens up the option to attach alternate SVG widgets to line endings.

Wouldn't cross over into the 3d plots very easily though.

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/marker-end

**Comment by demacdo** [comment]
Looking for the same thing, has anyone looked into this?
